docs/data/joy/main-features/automatic-adjustment/automatic-adjustment.md

```diff
@@ -1,58 +1,62 @@
 # Automatic adjustment
 
-<p class="description">Joy UI components adapt to one another when composed together to ensure the best look and feel.</p>
+<p class="description">Joy UI components adapt to one another to ensure consistency across your app without the need to micromanage your CSS.</p>
 
-When composing a UI of multiple, especially nested, components, oftentime you get small inconsistencies between them.
-That's probably when you'll be adjusting pixels by a few nudges here and there to get them just right.
-Over time, these small and arbitrary changes build up to become a problem.
+Joy UI components automatically adjust their styles and DOM structure relative to one another to ensure consistent sizing, spacing, and semantically appropriate HTML throughout your app.
```

Comment:
```suggestion
Joy UI components automatically adjust their styles and tag name relative to one another to ensure consistent sizing, spacing, and semantically appropriate HTML throughout your app.
```

Only one comment about "DOM structure". What Joy UI does is change the HTML tag to have appropriate semantics but never change the structure of the DOM.

I am not sure if my understanding of the word "DOM structure" is correct but I will rather be more specific using ["tag name"](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName) here.

Reply:
"The Document Object Model (DOM) is the data representation of the objects that comprise the structure and content of a document on the web." https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction

For me, the word "structure" refers to how the group of elements is structured. Joy UI will never change the structure.

Reply:
Interesting. I may be using the terminology incorrectly. When I consider, for example, replacing a `<div>` with a `<p>`, I think of that as changing the DOM structure—that is, the DOM tree has been altered. How would you describe this kind of change?

In your view, would you say that it's only a structural change when you're adding or subtracting nodes (i.e. adding a new `<p>` nested inside of a `<div>` instead of just swapping one for the other)?

Reply:
> In your view, would you say that it's only a structural change when you're adding or subtracting nodes (i.e. adding a new <p> nested inside of a <div> instead of just swapping one for the other)?

Yes, that's my perception. The word is quite blurry and I could not find any good references 😢. I'll leave you to decide on this then.

Reply:
Hmm, I can't find any good resources to support the terminology either way. What if we replace `## Style adjustments` and `## Structure adjustments` with `## CSS adjustments` and `## HTML adjustments`, since that's what these sections are really about anyway? And then I'll add your suggestion here for "tag name" instead of "DOM structure."